typing/cmt2annot.mli

```diff
@@ -0,0 +1,17 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                   Fabrice Le Fessant, INRIA Saclay                     *)
+(*                                                                        *)
+(*   Copyright 2018 Institut National de Recherche en Informatique et     *)
```

Comment:
This is anachronic: @lefessan didn't work for INRIA Saclay in 2018. I suggest just removing `Saclay`.

Reply:
Thanks, fixed.